packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -1123,7 +1168,11 @@ export class Select {
               small={small}
               menuId={menuId}
               open={this.open}
-              options={searchable ? this.filteredOptions : options}
+              options={
+                searchable
+                  ? this.filteredOptions
+                  : this.deduplicateOptions(options)
```

Comment:
just a comment on the efficiency of this. Currently this function will run every time the component is re-rendered. eg just opening & closing the menu causes it to run. would it be better to just run the code when the options are changed (eg a watch on the options)?

Reply:
I was unable to mutate the options object in the options watch function without causing a RangeError on the array and having to make this.options into a mutable prop - this might be a skill issue on my part. 
I guess maybe i'd need to create an internal "this.deduplidatedOptions" array that then gets filtered or rendered etc.?

Reply:
yes, that's what I was thinking, you could set\update this in the existing `watchOptionsHandler`